### PR TITLE
Creating A Data Model

### PR DESCRIPTION
**Pull request by aeh1707**
In this branch I have done the following:

- [x] Install Ruby on Rails framework.
- [x] Used database migration files to create the database schema.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,48 +215,31 @@ Feel free to check the [issues page](../../issues/).
...
+- Coding Partners
+- Inspiration
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (elisha2kyakpo1) [inline README.md:243]: - [ ] Let's add the license and add the link here. Avoid living broken links in the readme file.

File: README.md [hunk 2]
```diff
@@ -97,141 +93,119 @@ After you're finished please remove all the comments and instructions!
...
 
-> Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add a blog]**
+- [ ] **[Delete a blog]**
```
Comment (elisha2kyakpo1) [inline README.md:203]: - [ ] Let me understand the purpose of these features. We should only add necessary links/instructions or commands.

File: README.md [hunk 3]
```diff
@@ -60,18 +60,14 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [CodeRoamingInvestor] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**[your_project__name]** is a blog that explores the intersection of programming, travel, and investing through the eyes of Abdelhadi. As a passionate coder and traveler, Abdelhadi will share his experiences and insights on how to balance the world of technology with the desire to see new places and cultures. He will also share his knowledge on how to invest in technology and use it as a tool for financial success and freedom. This blog is for anyone looking for inspiration to combine their love for technology and travel and make smart investments for a fulfilling life.
```
Comment (elisha2kyakpo1) [inline README.md:65]: - [ ] Kindly add the name of your project where it says [your_project__name], you should remember that this is just a template that you should customize to suit the requirements at hand.
- [ ] The Microverse logo should be removed or replaced by your own.

File: db/migrate/20230120234349_creates_authors.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreatesAuthors < ActiveRecord::Migration[7.0]
+  def change
+    create_table :authors do |t|
```
Comment (elisha2kyakpo1) [inline db/migrate/20230120234349_creates_authors.rb:3]: - [ ] Let's follow the ERD diagram on table and column names, this table should be named users.

File: db/migrate/20230120234843_creates_posts.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class CreatesPosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: true, index: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20230120234843_creates_posts.rb:4]: - [ ] I'm happy to see that the name of a user will be called author as shown on the ERD diagram, but we should let Ruby know which table is an author we are referring to by explicitly adding `foreign_key: { to_table: 'users' }`

File: db/migrate/20230120235417_creates_comments.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+class CreatesComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :post, foreign_key: true, index: true
+      t.references :author, foreign_key: true, index: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20230120235417_creates_comments.rb:5]: - [ ] I'm happy to see that the name of a user will be called author as shown on the ERD diagram, but we should let Ruby know which table is an author we are referring to by explicitly adding `foreign_key: { to_table: 'users' }`

File: db/migrate/20230120235529_creates_likes.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+class CreatesLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :post, foreign_key: true, index: true
+      t.references :author, foreign_key: true, index: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20230120235529_creates_likes.rb:5]: - [ ] I'm happy to see that the name of a user will be called author as shown on the ERD diagram, but we should let Ruby know which table is an author we are referring to by explicitly adding `foreign_key: { to_table: 'users' }`

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @aeh1707, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @aeh1707 ,

Thanks for making the requested changes 👏  

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**